docs/pages/machine-workload-identity/machine-id/deployment/bound-keypair.mdx

```diff
@@ -0,0 +1,174 @@
+can be reconfigured at any time, and bots that expire or go offline can be
+recovered by making a server-side exemption without any client-side
+intervention.
+
+Refer to the [admin guide][guide] for further details on how this join method
+works.
+
+## Prerequisites
+
+{/* note: consider edition-prereqs-tabs.mdx include for v19; it is misleading due to the minor launch release */}
+
+- A running Teleport cluster version 18.1.0 or above.
+- The `tsh` and `tctl` clients
+- (!docs/pages/includes/tctl.mdx!)
+- This guide assumes the bot node has mutable persistent storage for internal
+  bot data. While it is possible to use Bound Keypair joining can on immutable
+  hosts (like CI runs), doing so will reduce security guarantees; see the
+  [admin guide][guide] for further information.
+
+## Step 1/5. Install `tbot`
+
+**This step is completed on the bot node.**
+
+First, `tbot` needs to be installed on the node that you wish to use Machine ID
+on.
+
+Download and install the appropriate Teleport package for your platform:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+## Step 2/5. Create a Bot
+
+**This step is completed on your local machine.**
+
+(!docs/pages/includes/machine-id/create-a-bot.mdx!)
+
+## Step 3/5. Create a join token
+
+**This step is completed on your local machine.**
+
+In this guide, we'll demonstrate joining a bot using a registration secret: this
+is a one-time use secret the bot can provide to Teleport to authenticate its
+first join. Once authenticated, the bot automatically generates a keypair and
+registers its public key with Teleport for use in all future join attempts.
+
+Create `token-example.yaml`:
+
+```yaml
+kind: token
+version: v2
+metadata:
+  # This name will be used in tbot's `onboarding.token` field.
+  name: example
+spec:
+  roles: [Bot]
+  # bot_name should match the name of the bot created earlier in this guide.
+  bot_name: example
+  join_method: bound_keypair
+  bound_keypair:
+    recovery:
+      mode: standard
+      limit: 1
+```
+
+Replace:
+- `example` in `spec.bot_name` with the name of the bot you created in the
+  second step
+
+For this example, we don't need to set any additional options for the bound
+keypair token. We've allowed a single recovery attempt, which will be used to
+allow the bot's initial join, and Teleport will generate a registration secret
+automatically when the token is created as we have not preregistered a public
+key to use.
+
+<Admonition type="tip" title="Onboarding and Recovery">
+To learn more about the various onboarding and recovery options, refer to the
+[Reference and Admin Guide][guide].
+</Admonition>
+
+Use `tctl` to apply this file:
+
+```code
+$ tctl create -f token-example.yaml
+```
+
+Next, retrieve the generated registration secret, which will be needed for the
+next step:
+```code
+$ tctl get token/example --format=json | jq -r '.[0].status.bound_keypair.registration_secret'
```

Comment:
This flow might be much easier to document (and user friendly) if we had a `tctl tokens add` command which created the token and output the registration secret.

Reply:
Hrm, I agree. I'll see if I can't throw something together.